src/DynamicEvaluator/Application/Checker/BrokenPhpFilesChecker/Dto/ViolationDto.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\Dto;
+
+use Upgrade\Application\Dto\ViolationDtoInterface;
+
+class ViolationDto implements ViolationDtoInterface
+{
+    /**
+     * @var array<string>
+     */
+    protected array $composerCommands;
+
+    /**
+     * @var array<\DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\Dto\FileErrorDto>
+     */
+    protected array $fileErrors;
+
+    /**
+     * @var string
+     */
+    protected string $hash;
+
+    /**
+     * @param array<string> $composerCommands
+     * @param array<\DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\Dto\FileErrorDto> $fileErrors
+     */
+    public function __construct(array $composerCommands, array $fileErrors)
+    {
+        $this->composerCommands = $composerCommands;
+        $this->fileErrors = $fileErrors;
+        $this->hash = sha1(
```

Comment:
DTO should have any logic. I got your idea. just easy way to compere violation. ok